### PR TITLE
fix swapped mem_usage/percent fields

### LINKED ISSUE #7160: Podman stats show swapped memory usage and percent on Ubuntu 18
**Is this a BUG REPORT or FEATURE REQUEST? (leave only one on its own line)**

/kind bug

**Description**

On Ubuntu 18.04:

Checking `podman stats` for a container has `memory_usage` and `memory_percent` switched. This is causing some monitoring issues and 100% consistent. I have not verified on other OS.

**Steps to reproduce the issue:**

1. Started a container with podman: `/usr/bin/podman run --name channelrouter-1596133146553893 --log-opt path=/tmp/channelrouter-1596133146553893.log -d channel-router:v1`

2. Ran `podman stats --no-stream --format json`

**Describe the results you received:**

```
[
 {
  "id": "156770243e72",
  ...
  "mem_usage": "3.19%",
  "mem_percent": "66.59MB / 2.09GB",
  ...
 }
]
```

**Describe the results you expected:**

```
[
 {
  "id": "156770243e72",
  ...
  "mem_usage": "66.59MB / 2.09GB",
  "mem_percent": "3.19%",
  ...
 }
]
```

**Additional information you deem important (e.g. issue happens only occasionally):**

**Output of `podman version`:**

```
Version:      2.0.3
API Version:  1
Go Version:   go1.14.4
Built:        Wed Dec 31 16:00:00 1969
OS/Arch:      linux/amd64
```

**Output of `podman info --debug`:**

```
host:
  arch: amd64
  buildahVersion: 1.15.0
  cgroupVersion: v1
  conmon:
    package: 'conmon: /usr/libexec/podman/conmon'
    path: /usr/libexec/podman/conmon
    version: 'conmon version 2.0.18, commit: '
  cpus: 2
  distribution:
    distribution: ubuntu
    version: "18.04"
  eventLogger: file
  hostname: sparker1.host.example.com
  idMappings:
    gidmap: null
    uidmap: null
  kernel: 4.15.0-101-generic
  linkmode: dynamic
  memFree: 459411456
  memTotal: 2089709568
  ociRuntime:
    name: runc
    package: 'runc: /usr/sbin/runc'
    path: /usr/sbin/runc
    version: 'runc version spec: 1.0.1-dev'
  os: linux
  remoteSocket:
    path: /run/podman/podman.sock
  rootless: false
  slirp4netns:
    executable: ""
    package: ""
    version: ""
  swapFree: 15996796928
  swapTotal: 15997071360
  uptime: 17m 9.97s
registries:
  ...
  search:
  - docker.io
  - quay.io
store:
  configFile: /etc/containers/storage.conf
  containerStore:
    number: 1
    paused: 0
    running: 1
    stopped: 0
  graphDriverName: overlay
  graphOptions: {}
  graphRoot: /var/lib/containers/storage
  graphStatus:
    Backing Filesystem: xfs
    Native Overlay Diff: "true"
    Supports d_type: "true"
    Using metacopy: "false"
  imageStore:
    number: 1
  runRoot: /var/run/containers/storage
  volumePath: /var/lib/containers/storage/volumes
version:
  APIVersion: 1
  Built: 0
  BuiltTime: Wed Dec 31 16:00:00 1969
  GitCommit: ""
  GoVersion: go1.14.4
  OsArch: linux/amd64
  Version: 2.0.3
```

**Package info (e.g. output of `rpm -q podman` or `apt list podman`):**

```
podman/unknown,now 2.0.3~1 amd64 [installed]
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**

### PR DESCRIPTION
**Pull request by Luap99**
Correct the wrong field assignment in `podman stats --format=json`.

Fixes #7160

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM
@vrothberg @giuseppe @TomSweeneyRedHat @ashley-cui @QiWang19 @baude @jwhonce PTAL

@Luap99 I wrote the same PR, so I guess this works for me.

**Comment by ashley-cui** [comment]
LGTM

**Comment by edsantiago** [comment]
@Luap99 this is a known hiccup in CI; try rebasing on latest master

**Comment by Luap99** [comment]
@edsantiago OK, lets see now.

File: cmd/podman/containers/stats.go [hunk 1]
```diff
@@ -230,8 +230,8 @@ func outputJSON(stats []*containerStats) error {
 			Id:         j.ID(),
 			Name:       j.Name,
 			CpuPercent: j.CPUPerc(),
-			MemUsage:   j.MemPerc(),
-			MemPerc:    j.MemUsage(),
+			MemUsage:   j.MemUsage(),
+			MemPerc:    j.MemPerc(),
```
Comment (TomSweeneyRedHat) [inline cmd/podman/containers/stats.go:234]: ooops!  nice fix/catch

**Pull request merged**